src/utils/DeleteMultiMessages.ts

```diff
@@ -5,15 +5,16 @@ export const deleteMultiMessages = async (
     channel: GuildTextBasedChannel | DMChannel | PartialDMChannel,
     messages: Collection<string, Message<boolean>>
 ) => {
-    const threads = messages.filter(m => m.hasThread).map(m => m.thread as AnyThreadChannel);
-    const [recentMessages, oldMessages] = messages.partition(
+    const nonSystemMessages = messages.filter(message => !message.system);
+    const threads = nonSystemMessages.filter(m => m.hasThread).map(m => m.thread as AnyThreadChannel);
+    const [recentMessages, oldMessages] = nonSystemMessages.partition(
```

Comment:
https://discord.js.org/docs/packages/discord.js/14.14.1/Message:Class#bulkDeletable
discord.jsのアップデートでbulkDeleteで消せるかを判定するフィールドが追加されているのでそれに直したい
```suggestion
    const [recentMessages, oldMessages] = nonSystemMessages.partition(m=>m.bulkDeletable);
```
みたいな感じで今は行けると思う